apps/confidential/confidential.go

```diff
@@ -315,16 +317,21 @@ func New(authority, clientID string, cred Credential, options ...Option) (Client
 	if err != nil {
 		return Client{}, err
 	}
-
+	region := os.Getenv("MSAL_FORCE_REGION")
```

Comment:
naming: call it smth more accurage, `region` is pretty general. autoEnabledRegion or smth